api/cuckoo_service.py

```diff
@@ -8,19 +8,23 @@
 from requests.exceptions import Timeout
 from datetime import datetime
 from http import HTTPStatus
+from dotenv import load_dotenv
 
+from flask import current_app
 from .log import LogManager
 from .model_service import start_model_monitor, upload_to_model
 
 CUCKOO_URL = 'http://140.124.181.155'
 PORT = '1337'
-HEADERS = {"Authorization": "Bearer YCjQF5fx4Ladj09Hf5ZApg"}
+load_dotenv() 
+HEADERS = os.getenv('HEADERS')
+HEADERS = json.loads(HEADERS) 

```

Comment:
As I said before, it's not good to have the global const variable. It should store in config of Flask.
It will be easy to modified and maintaince the configuration.